h/presenters.py

```diff
@@ -63,11 +63,34 @@ def __init__(self, annotation):
         else:
             self.document = None
 
+    def _get_selection(self):
+        selectors = self.annotation.target_selectors
+        for selector in selectors:
+            if "exact" in selector:
+                return selector["exact"]
+
     @property
     def uri(self):
         return jinja2.escape(self.annotation.target_uri)
 
     @property
+    def text_rendered(self):
+        """The body text of this annotation.
+
```

Comment:
The empty new line in the comment is not consistent with other single-line API comments in the file. Should just be `"""One line description."""`